components/CarouselWithImages.vue

```diff
@@ -0,0 +1,58 @@
+<script setup>
+const props = defineProps({
+  images: {
+    type: Array,
+    required: true
+  }
+})
+</script>
+
+<template>
+  <v-carousel v-if="props?.images && props.images?.length" 
```

Comment:
After props destructuring in <script setup>, simplifying the v-if condition should work?

v-if="images && images.length"